server/src/main/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinator.java

```diff
@@ -108,7 +110,7 @@ public IndexerSQLMetadataStorageCoordinator(
     this.connector = connector;
   }
 
-  enum DataSourceMetadataUpdateResult
+  enum DataStoreMetadataUpdateResult
```

Comment:
Previously, in IndexerSQLMetadataStorageCoordinator we only update the datasource table, hence the result enum is named DataSourceMetadataUpdateResult. However, now the IndexerSQLMetadataStorageCoordinator have methods to update both the datasource table and the segment table in metadata store. Instead of having two enum which are pretty much the same, I just rename this enum so that it can be use for handling the result of updating the metadata store...no matter which tables.